sdk/keyvault/keyvault-admin/test/public/utils/authentication.ts

```diff
@@ -41,11 +36,13 @@ export async function authenticate(
     sanitizerOptions: {
       generalSanitizers: [
         {
+          // eslint-disable-next-line no-useless-escape
           target: `keyvault_name\.[a-z-]+\.azure[a-z-]*\.net`,
```

Comment:
I think the lint rule might be on to something actually:

```suggestion
          target: `keyvault_name\\.[a-z-]+\\.azure[a-z-]*\\.net`,
```

I'm guessing the regex was working OK previously just because the unescaped `.` is just matching a single character (in this case, an actual "."). So it's been working and we didn't catch it until now.